packages/lib/src/playground/bar-sparkline.vue

```diff
@@ -1,23 +1,97 @@
 <template>
-  <LumeBarChart v-bind="$props" />
+  <lume-chart
+    v-bind="$props"
+    :options="options"
+    :style="{ width: `${width}px` }"
+    class-list="bar-sparkline"
+  >
+    <template #groups="groupProps">
+      <lume-bar-group v-bind="groupProps" />
+
+      <g>
+        <rect
+          v-for="(_l, index) in groupProps.labels.length"
+          v-bind="getBarCapProps(index, groupProps)"
+          :key="`cap_${index}`"
+        />
+      </g>
+    </template>
+  </lume-chart>
 </template>
 
-<script lang="ts">
-import { defineComponent } from 'vue';
+<script setup lang="ts">
+import { computed } from 'vue';
 
-import LumeBarChart from '@/components/charts/lume-bar-chart';
+import LumeChart from '@/components/core/lume-chart';
+import LumeBarGroup from '@/components/groups/lume-bar-group';
 
 import { withChartProps } from '@/composables/props';
 
-export default defineComponent({
-  components: { LumeBarChart },
-  props: {
-    ...withChartProps(),
+import type { ChartOptions } from '@/types/options';
+
+const props = defineProps({
+  ...withChartProps(),
+  barWidth: {
+    type: Number,
+    required: true,
   },
-  setup() {
-    return {};
+  gap: {
+    type: Number,
+    required: true,
   },
 });
+
+const { barWidth, gap } = props;
```

Comment:
Better to do `toRefs(props)`. `barWidth` and `gap` doesn't seem to be reactive in the deployment preview :)

Reply:
Good point - fixed